distribution/docs/src/main/resources/_contents/_security-framework/security-sts-client-config-contents.adoc

```diff
@@ -0,0 +1,84 @@
+
+==== Security STS Client Config
+
+The ${ddf-security} STS Client Config bundle keeps track and exposes configurations and settings for the CXF STS client.
+This client can be used by other services to create their own STS client.
+Once a service is registered as a watcher of the configuration, it will be updated whenever the settings change for the sts client.
+
+===== Installing Security STS Client Config
+
+This bundle is installed by default.
+
+===== Configuring Security STS Client Config
+
+. Navigate to the ${admin-console}.
+. Select *${ddf-security}* Application.
+. Select *Configuration* tab.
+. Select *Security STS Client*.
+
+.Security STS Client Config Settings
+[cols="1,2,2" options="header"]
+|===
+
+|Configuration Name
+|Default Value
+|Additional Information
+
+|SAML Assertion Type
+|SAML v2.0
+|The version of SAML to use. Most services require SAML v2.0. Changing this value from the default could cause services to stop responding.
+
+|SAML Key Size
+|256
+|The key type to use with SAML. Most services require Bearer. Changing this value from the default could cause services to stop responding.
```

Comment:
I think we may have left part of the description out or we left it out on the metatype that this was pulled from. The second sentence should be "Most services require Bearer and this value will not impact Bearer assertions." or something like that.